python/res/fm/rms/rms_run.py

```diff
@@ -155,7 +145,10 @@ def wait(self, child_pid):
             )
 
     def exec_rms(self, exec_env):
-        args = [
+        args = []
+        if self.config.wrapper is not None:
+            args = [self.config.wrapper]
+        args += [
```

Comment:
or like this:
```suggestion
        args = []
        if self.config.wrapper is not None:
            args.append(self.config.wrapper)
        args.extend([
```

NB: Missing ) at the end